### PR TITLE
Hide popup label to prevent broken layout

### PR DESCRIPTION
**Pull request by regiskuckaertz**
This will only happen when the viewport is between `leftCol` and `wide`.

Before (top left):
picture 3

After:
picture 1

### CODE REVIEW

**Comment by jbreckmckye** [comment]
Would we actually want to hide the about link in this instance? Perhaps it should just appear on a second line without the left margin.

**Comment by regiskuckaertz** [comment]
Alright, doing that then.

**Comment by regiskuckaertz** [comment]
That way.

picture 1

File: static/src/stylesheets/module/commercial/_paidfor.scss [hunk 1]
```diff
@@ -31,6 +31,12 @@
         overflow: visible;
     }
 
+    .paidfor-meta__more > .popup__toggle {
```
Comment (uplne) [inline static/src/stylesheets/module/commercial/_paidfor.scss]: You know that this is one of the most expensive selectors

**Comment by uplne** [comment]
:+1:

File: static/src/stylesheets/module/commercial/_paidfor.scss (hunk 1 above)
Comment (jbreckmckye) [inline static/src/stylesheets/module/commercial/_paidfor.scss]: Surely, that's less expensive than `.foo .bar`?

**Pull request merged**